### PR TITLE
add performed tests runner

### PR DESCRIPTION
**Pull request by maekke**
add the missing pieces to get performed tests running

- changed the csv file names to use English names
- moved a few pieces to a new db_common.py file
- added a 'total tests' column for the cantons providing only this number and the positivity rate

### CODE REVIEW

File: .github/workflows/run_tests_scraper.yml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+name: Run district scrapers
```
Comment (metaodi) [inline .github/workflows/run_tests_scraper.yml:1]: This should be something like "Run tests scraper"

Comment (maekke) [reply .github/workflows/run_tests_scraper.yml]: You're right! Should be fixed in https://github.com/openZH/covid_19/pull/1342/commits/f7b01d50ef3f5580f657dc40efc69ec2b5d17c65

**Comment by metaodi** [comment]
I just had a phone call with @andreasamsler to discuss this PR

Notes:

- positivity_rate is missing from the CSV
- if possible we should not to calculations on our own, but rather provide the data by the cantons (so either have `positive_tests` and `negative_tests` OR have `total_tests` and `positivity_rate`).  Depending on what numbers the cantons provide.
- The field `url` should be renamed to `source` to be consistent with the other CSVs

Other than that, this is good to go!

**Comment by maekke** [comment]
> I just had a phone call with @andreasamsler to discuss this PR
> 
> Notes:
> 
>     * positivity_rate is missing from the CSV
> 
>     * if possible we should not to calculations on our own, but rather provide the data by the cantons (so either have `positive_tests` and `negative_tests` OR have `total_tests` and `positivity_rate`).  Depending on what numbers the cantons provide.
> 
>     * The field `url` should be renamed to `source` to be consistent with the other CSVs
> 
> 
> Other than that, this is good to go!

Alright, we're getting closer then!

Thanks for spotting the missing positivity_rate, should be added. Also renamed the url to source.

I've updated all scrapers to avoid any calculations, but add what is available from the source.

**Comment by metaodi** [comment]
Thanks a lot @maekke !

**Pull request merged**